module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,12 @@
+#include "sort.hpp"
+
+#include <functional>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> mail) {
+    std::deque<std::string> vec;
+    mail.sort(std::greater<std::string>());
+    for (const auto& el : mail) {
+        vec.push_front(el);
+    }
+    return vec;
```

Comment:
instead of pushing elements to created vec, you can create vec after sorting of list and use this sorted list in a vector: `container(first_container.begin(), first_container.end())`

Reply:
+1